x/ibc/testing/solomachine.go

```diff
@@ -32,27 +33,31 @@ func NewSolomachine(t *testing.T, clientID string) *Solomachine {
 	privKey := ed25519.GenPrivKey()
 
 	return &Solomachine{
-		t:          t,
-		ClientID:   clientID,
-		PrivateKey: privKey,
-		PublicKey:  privKey.PubKey(),
-		Sequence:   1,
-		Time:       10,
+		t:           t,
+		ClientID:    clientID,
+		PrivateKey:  privKey,
+		PublicKey:   privKey.PubKey(),
+		Sequence:    1,
+		Time:        10,
+		Diversifier: "", // TODO: use custom diversifier?
```

Comment:
Won't this fail `ValidateBasic` because it's blank?

Perhaps use `"testing"`

Reply:
the diversifier is allowed to be empty just not blank with spaces